steps/monitoring-apps-permissions/1.0.0/step.yml

```diff
@@ -0,0 +1,117 @@
+title: Mobile apps permissions monitoring
+summary: |
+  Check and monitor mobile applications permissions
+description: |
+  Check mobile applications permissions and compare with configured values to be warned when new permissions have been added
+
+  ### Usage
+
+  To give to our step the informations about the expected values, you have:
```

Comment:
In English "information" is a singulare tantum.

Reply:
Fixed in #2875